src/main/java/constants/Operator.java

```diff
@@ -0,0 +1,33 @@
+package constants;
+
+import java.util.Arrays;
+
+public enum Operator {
+      PLUS("PLUS", "+")
+    , MINUS("MINUS", "-")
+    , MULTIPLICATION("MULTIPLICATION", "*")
+    , DIVISION("DIVISION", "/");
```

Comment:
```suggestion
      PLUS("+", new Plus())
    , MINUS("-", new Minus())
    , MULTIPLICATION("*", new Multiplication())
    , DIVISION("/", new Division());
    
    public int operate(num1, num2) {
        this.operator.calculate(num1, num2)
    }
```
만들어둔 연산자들을 위 방식으로 만들어보면 어떨까요?